src/test/java/net/snowflake/ingest/streaming/internal/InsertRowsBenchmarkTest.java

```diff
@@ -36,7 +40,7 @@ public class InsertRowsBenchmarkTest {
 
   @Setup(Level.Trial)
   public void setUpBeforeAll() {
-    client = new SnowflakeStreamingIngestClientInternal<ParquetChunkData>("client_PARQUET");
+    client = new SnowflakeStreamingIngestClientInternal<ParquetChunkData>("client_PARQUET", false);
```

Comment:
please add a code comment here with `SNOW-1490151` in the comment so that whenever that JIRA is fixed we have an easy way of finding all the places that need fixes.

Reply:
Added.